extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/common/KubernetesPeonClient.java

```diff
@@ -69,12 +85,14 @@ public Pod launchPeonJobAndWaitForStart(Job job, long howLong, TimeUnit timeUnit
                          }, howLong, timeUnit);
       long duration = System.currentTimeMillis() - start;
       log.info("Took task %s %d ms for pod to startup", jobName, duration);
+      emitK8sPodMetrics(job, "peon/startup/time", duration);
```

Comment:
I think it would be good to preface the metric with `k8s` so that it is clear that the metric only applies to peons started by the kubernetes task runner.

I also like adding the unit to the end of the metric name to make it easier for operators to understand what unit the metric is reported in.

```suggestion
      emitK8sPodMetrics(job, "k8s/peon/startup/timeMillis", duration);
```

Reply:
I like the idea of prefix with k8s, will do that. But I saw other metrics are not end with unit such as `task/run/time` wonder whether we should align with the existing ones.